trieste/acquisition/optimizer.py

```diff
@@ -165,34 +170,29 @@ def optimize_continuous(space: Box, target_func: AcquisitionFunction) -> TensorT
         )  # [num_optimization_runs]
         initial_points = tf.gather(trial_search_space, top_k_indicies)  # [num_optimization_runs, D]
 
-        if sigmoid:  # use scipy's L-BFGS optimizer with a sigmoid transform
-            bijector = tfp.bijectors.Sigmoid(low=space.lower, high=space.upper)
-            opt_kwargs = {}
-        else:  # use scipy's L-BFGS-B optimizer
-            bijector = tfp.bijectors.Identity()
-            opt_kwargs = {"bounds": spo.Bounds(space.lower, space.upper)}
-
-        variable = tf.Variable(bijector.inverse(initial_points[0:1]))  # [1, D]
+        variable = tf.Variable(initial_points[0:1])  # [1, D]
 
         def _objective() -> TensorType:
-            return -target_func(bijector.forward(variable[:, None, :]))  # [1]
+            return -target_func(variable[:, None, :])  # [1]
 
-        def _perform_optimization(starting_point: TensorType) -> OptimizeResult:
-            variable.assign(bijector.inverse(starting_point))  # [1, D]
+        def _perform_optimization(starting_point: TensorType, bounds: spo.Bounds) -> OptimizeResult:
+            opt_kwargs = {"bounds": bounds}
+            variable.assign(starting_point)  # [1, D]
             return gpflow.optimizers.Scipy().minimize(_objective, (variable,), **opt_kwargs)
```

Comment:
Since the kwargs aren't defined at runtime, you can just specify them directly

```suggestion
            variable.assign(starting_point)  # [1, D]
            return gpflow.optimizers.Scipy().minimize(_objective, (variable,), bounds=bounds)
```